src/components/pages/order/main/Admin/AdminBloc.js

```diff
@@ -0,0 +1,22 @@
+import { useContext } from "react";
+import styled from "styled-components";
+import OrderContext from "../../../../../context/OrderContext";
+import AdminPanel from "./AdminPanel";
+import AdminTabs from "./AdminTabs";
+
+export default function AdminBloc() {
```

Comment:
```suggestion
export default function Admin() {
```
tout simplement 😄 , ou quelque chose de plus explicite peut-être

Reply:
AdminBloc pourrait faire penser à la partie en dessous, donc c'est vrai que Admin serait plus adapté ici, au même titre que Menu ou Basket qui restent général pour tout contenir.